_data/accessibility-tests/tooltip.yml

```
@@ -0,0 +1,98 @@
+title: Tooltip
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to communicate the purpose of the tooltip.
+    summary_additional: |
+      When you view the tooltip, the text, and never color alone, convey its purpose.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Appropriate color contrast between tooltip text and background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the background and text is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Appropriate color contrast between tooltip and page background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the tooltip component itself and any adjacent background content is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+  - summary: You can hover over tooltip content.
+    summary_additional: When you move the pointer over an active tooltip, the tooltip stays open.
+    test_status: fail
+    github_issue_number: 5919
+    github_issue_repo: uswds
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+  - summary: Active tooltips stay open until they are dismissed.
+    summary_additional: Active tooltips close only when you move focus away from the tooltip trigger or press the `escape` key.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the tooltip or other content.
+    summary_additional: When you zoom to 200%, the tooltip content remains visible and does not overlap with other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Tooltip remains visible and functional when screen is magnified.
+    summary_additional: When you zoom to 200%, you can still read content in the tooltip and surrounding area easily.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Active tooltip content can be closed with a mouse or keyboard.
+    summary_additional: When you open a tooltip with a mouse, you can use the `escape` key to close it.
+    test_status: fail
+    github_issue_number: 5909
+    github_issue_repo: uswds
+    test_type: keyboard
```

@amycole501 @alex-hull @sarah-sch 

I think this should be re-written a bit since it doesn't seem to be about closing it with a mouse. I borrowed some language from [1.4.13](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) for this suggestion. I also wonder if this should be moved to "general". Let me know what you think. 

```suggestion
  - summary: Active tooltips can be dismissed without moving the mouse or keyboard focus. 
    summary_additional: When you open a tooltip with a mouse, you can use the `escape` key to close it.
    test_status: fail
    github_issue_number: 5909
    github_issue_repo: uswds
    test_type: general
```

---

@amyleadem - the language looks good to me. I will leave it to @amycole501 and @alex-hull to confirm that the information is correct.

---

The new wording does make for a simpler-to-follow test. My only slight concern is that the person knows that even if they hover over the element to activate the tooltip, the tooltip should be persistent even if you move the mouse. BUT, I think that your wording change still incorporates that into the test so we are probably good.

Also, good catch on moving this to general since both a keyboard and a mouse are involved.

---

I think this is a good note. I tried adding a bit of clarity to the summary in [bdb277a](https://github.com/uswds/uswds-site/pull/2646/commits/bdb277aeb9f784c271fb821574458f0538571e48).

"Active tooltips can be dismissed without moving the mouse or keyboard focus." --> 
"Active tooltips can be dismissed without moving the mouse or keyboard focus _away from the tooltip trigger_."

Let me know what you think!